app/jobs/export/labels_job.rb

```diff
@@ -16,13 +16,18 @@ def initialize(format, user_id, people_ids, group_id, options)
   end
 
   def people
-    @people ||= Person.where(id: @people_ids)
+    @people ||= Person.where(id: @people_ids).order(order_statement)
   end
 
   def group
     @group ||= Group.find(@group_id)
   end
 
+  def order_statement
+    # retain order of @people_ids but allow override in wagons
```

Comment:
We mostly have the comments above the method, so...

```suggestion
  # retain order of @people_ids but allow override in wagons
  def order_statement
```